ben-spier.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Timeline</title>
+    <meta http-equiv="content-type" content="text/html" charset="UTF-8">
+    <link rel="stylesheet" href="ben-spier.css">
+  </head>
+
+  <body>
+    <h1>Brief History of CSS</h1>
+    <ul>
+      <li id="firstListItem"><div class="date">1994</div>: The first work on style sheets started at CERN.</li> <!--not sure if right to use div element to create class but it worked!-->
```

Comment:
You are forcing the `<div>` to be inline via CSS, but there is already an element that does what you need :nerd_face:

Reply:
Which element would that be? 

I thought style considerations should be left to CSS? 🙃